### PR TITLE
add basic slides to k6-learn

### PR DESCRIPTION
**Pull request by mdcruz**
This PR adds basic slides support using reveal.js. Same content from the modules has been re-used.

To view the slides locally in your terminal, run the following command:

```
npm install
npm run slides
```

https://github.com/grafana/k6-learn/assets/10811379/2f7ea18d-7d04-444d-904d-8cb69aa4257f

### CODE REVIEW

File: slides/04-getting-started-with-k6-oss/slides.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# Get started with k6 OSS
+
+---
+
+## Installation
+
+https://k6.io/docs/getting-started/installation/ 
+
+![k6 installation page](../../images/installation-page.png)
```
Comment (javaducky) [inline slides/04-getting-started-with-k6-oss/slides.md:9]: This page is strange because I cannot scroll to see the full image.

**Review by javaducky — APPROVED** [review]
LGTM! The only issue was related to a vertically large image of the installation instructions for k6 itself.

File: README.md [hunk 2]
```diff
@@ -14,6 +14,19 @@ This repo contains resources for:
 
 Consider running a workshop for k6. Below is an outline of what that workshop could look like, as well as modules you could use for each topic. Feel free to take these and include the parts most relevant to you!
 
+We have also created built-in slides for you, which you are free to edit. For a more custom slides, please fork this repo and make the edits accordingly how you want the workshop to be structured.
```
Comment (heitortsergent) [inline README.md:17]:
```suggestion
We have also created built-in slides for you, which you are free to edit. To customize the slides, please fork this repo and make edits according to how you want the workshop to be structured.
```

File: slides/03-load-testing/slides.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+**Performance testing != Load testing**
+
+---
+
+## Common test parameters
+
+_Test parameters_ include the distribution, shape, and pattern of the load.
+
+- **Virtual users (VUs).**
```
Comment (heitortsergent) [inline slides/03-load-testing/slides.md:21]:
```suggestion
- **Virtual users (VUs)**  
```

File: slides/03-load-testing/slides.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+## Common test parameters
+
+_Test parameters_ include the distribution, shape, and pattern of the load.
+
+- **Virtual users (VUs).**  
+- **Iterations**
+- **Throughput**
+- **User flows**
+- **Load profile**.
```
Comment (heitortsergent) [inline slides/03-load-testing/slides.md:25]:
```suggestion
- **Load profile**
```

File: slides/03-load-testing/slides.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+## Load test types
+
+### Shakeout test
+
+![](../../images/scenarios-shakeout.png) 
+<!-- .element class="stretch" -->
+
+---
+
+### Average Load Test
```
Comment (heitortsergent) [inline slides/03-load-testing/slides.md:47]:
```suggestion
### Average load test
```

File: slides/03-load-testing/slides.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+---
+
+### Stress test
+
+![](../../images/test-scenario-stress.png)
+<!-- .element class="stretch" -->
+
+---
+
+### Soak or Endurance Test
```
Comment (heitortsergent) [inline slides/03-load-testing/slides.md:61]:
```suggestion
### Soak or endurance test
```

File: slides/03-load-testing/slides.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+
+![](../../images/test-scenario-soak.png)
+<!-- .element class="stretch" -->
+
+---
+
+### Spike Test
```
Comment (heitortsergent) [inline slides/03-load-testing/slides.md:68]:
```suggestion
### Spike test
```

File: slides/03-load-testing/slides.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+
+![](../../images/test-scenario-spike-test.png)
+<!-- .element class="stretch" -->
+
+---
+
+### Breakpoint Test
```
Comment (heitortsergent) [inline slides/03-load-testing/slides.md:75]:
```suggestion
### Breakpoint test
```

File: slides/04-getting-started-with-k6-oss/slides.md (hunk 1 above)
Comment (mdcruz) [reply slides/04-getting-started-with-k6-oss/slides.md:9]: I'll change this image because it is actually cropped 😅

**Pull request merged**